CONTRIBUTING.md

```diff
@@ -15,9 +15,10 @@ You can help us **by submitting bug reports or fixing bugs** in the [QGIS bug tr
 If you wish to contribute patches you can:
 
 1. [fork the project](https://help.github.com/forking/)
-1. make your changes
-1. commit to your repository
-1. and then [create a pull request](https://help.github.com/articles/creating-a-pull-request-from-a-fork/).
+2. Install the pre-commit hook: `ln -s ../../scripts/prepare_commit.sh .git/hooks/pre-commit`
```

Comment:
```suggestion
2. Install the pre-commit hook: `pre-commit install --install-hooks`
```
Will do the job by using the `.pre-commit.yaml` file. The yaml file already reference the `scripts/prepare_commit.sh`

Reply:
I don't have a "pre-commit" command, which package should contain it ?

Reply:
I'm trying to make contributors life easier, so the least things they need to install, the better

Reply:
For the record, I think this is a bad idea to not advertise `pre-commit`